pages/plugins/[plugin].js

```diff
@@ -11,3 +11,8 @@ export default function specifyPlugin() {
   }
   return null
 }
+
+specifyPlugin.getInitialProps = () => {
+  console.log("Is this enough to make this dynamic?")
```

Comment:
Turns out we don't even need it to do anything: https://github.com/buildkite/site/pull/573/commits/f42d965f164a346466e6f6bf281b72c1f52d677f